test/tools/libtesteth/TestSuite.h

```diff
@@ -20,19 +20,32 @@
 
 #pragma once
 #include <test/tools/libtesteth/JsonSpiritHeaders.h>
+#include <boost/filesystem/path.hpp>
 
 namespace dev
 {
 namespace test
 {
 
-class TestSute
+class TestSuite
 {
 public:
-	virtual ~TestSute() {}
+	virtual ~TestSuite() {}
+
+	//Main test executive function. should be declared for each test suite. it fills and run the test .json file
```

Comment:
"declared" is not quite correct (declaration is only the signature of the function), I'd say "implemented". (Also it's kind of clear from the fact that its pure virtual function, so maybe don't mention it at all)

Reply:
typo: "run" => "runs"

Reply:
Space after `//` would make it look nicer
Also, better start sentences with capital letter.